pkg/sysinfo/hugepages.go

```diff
@@ -72,6 +72,10 @@ func HugepagesForNode(hnd Handle, nodeID int) ([]Hugepages, error) {
 		return nil, err
 	}
 	for _, entry := range entries {
+		if !entry.IsDir() {
```

Comment:
Is this deserve log? genuinely asking

Reply:
actually yes, but not erroring out. Adding.